RelativityDev.RelativityTestHelpers.nuspec

```diff
@@ -16,6 +16,7 @@
 	<dependencies>
       <group targetFramework=".NETFramework4.6.2">
         <dependency id="Relativity.DataExchange.Client.SDK" version="1.4.0" />
+        <dependency id="Relativity.ObjectManager" version="10.3.170.1" />
```

Comment:
do we still need this specific version of OM? I thought we were going to use the Rest version of the Kepler services.

Reply:
We decided to not use REST endpoints (at least not yet) on this project.  Could be a potential update in the future though